### PR TITLE
Created SupplierProfilePage

### PR DESCRIPTION
**Pull request by rares2k**
Created the Supplier Profile Page, the appbar still needs to be changed, but the rest of the page is fine.

### CODE REVIEW

File: lib/main.dart [hunk 1]
```diff
@@ -20,7 +21,7 @@ class _MyAppState extends State<MyApp> {
   @override
   Widget build(BuildContext context) {
     return MaterialApp(
-        home: LoginPage()
+        home: SupplierProfilePage()
```
Comment (CimpeanAndreea) [inline lib/main.dart]: Revert this, will be handled when navigation is written properly

File: lib/supplierprofile/supplierprofilepage.dart [hunk 2]
```diff
@@ -0,0 +1,452 @@
...
+                              ),
+                            ]))
+                  ],
+                ),
+              ),
+            ),
+            Container(
+              decoration: BoxDecoration(
+                  image: DecorationImage(
+                image: AssetImage('assets/bluegradient.png'),
```
Comment (BuildNowDpit) [inline lib/supplierprofile/supplierprofilepage.dart]: Use constants from CONSTANTS FILE!!!!!!!!!!!!!!!!!!

File: lib/supplierprofile/AppBar.dart [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+AppBar buildAppBar(BuildContext context) {
+
+  Icon search_icon = Icon(Icons.search_outlined);
+  Widget search = Text("");
+
+  return AppBar(
+    flexibleSpace: Container(
+      decoration: BoxDecoration(
+          image: DecorationImage(
+              image: AssetImage('assets/bluegradient.png'),
```
Comment (CimpeanAndreea) [inline lib/supplierprofile/AppBar.dart]: Pretty sure we already have a constant for this

File: lib/supplierprofile/supplierprofilepage.dart [hunk 4]
```diff
@@ -0,0 +1,452 @@
...
+  @override
+  State<SupplierProfilePage> createState() => _SupplierProfilePage();
+}
+
+class _SupplierProfilePage extends State<SupplierProfilePage> {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+        appBar: buildAppBar(context),
+        backgroundColor: Color(0xE7052C7),
```
Comment (CimpeanAndreea) [inline lib/supplierprofile/supplierprofilepage.dart]: Define a constant for this if it is not already defined

File: lib/supplierprofile/supplierprofilepage.dart (hunk 2 above)
Comment (CimpeanAndreea) [inline lib/supplierprofile/supplierprofilepage.dart]: Use the constant for this

**Review by BuildNowDpit — COMMENTED** [review]
CONSTANTS

**Comment by CimpeanAndreea** [comment]
Make sure to resolve the merge conflict between this and the master branch

**Pull request merged**